tests_new/integration_tests/core/environment/queries.py

```diff
@@ -61,10 +61,14 @@ def create_environment(client, name, group, organizationUri, awsAccountId, regio
                 'region': region,
                 'description': 'Created for integration testing',
                 'tags': tags,
+                'type': 'IntegrationTesting',
                 'parameters': [
                     {'key': 'notebooksEnabled', 'value': 'true'},
+                    {'key': 'dashboardsEnabled', 'value': 'true'},
+                    {'key': 'mlStudiosEnabled', 'value': 'false'},
```

Comment:
Note: need to provide env params to create env otherwise they do not get created and then causes issues with `set_env_params(...)` util for testing

Reply:
All env params?

Reply:
I wasn't aware of this limitation, would it make sense to adjust the `set_env_params` accordingly?

Reply:
`set_env_params` takes all the `env.parameters` returned from backend and if any of the values differ - sets the new value and updates the environment via API request

As long as the create_environment input had all the parameters as part of the input the functionality of `set_env_params` should be good as is - unless the expectation is for `set_env_params` to also identify and add env parameters to the environment if they do not exist already?

Reply:
@petrkalos - I think we are good as is but please confirm

Will merge this PR Monday unless you or others suggest additional changes